server/task/src/main/java/org/apache/james/task/MemoryTaskManagerWorker.java

```diff
@@ -35,6 +33,8 @@
 import reactor.core.publisher.Mono;
 
 public class MemoryTaskManagerWorker implements TaskManagerWorker {
+
+
```

Comment:
Should not it be only one?

Reply:
I can change that but will not wait for the build to be green if you agree